pkg/operator/ceph/config/monstore.go

```diff
@@ -244,11 +244,16 @@ func (m *MonStore) setAll(clientName string, settings map[string]string) ([]stri
 		rook.TerminateFatal(errors.Wrapf(err, "failed to write config file"))
 	}
 
+	outFilePath := assimilateConfPath.Name() + ".out"
 	defer func() {
 		err := os.Remove(assimilateConfPath.Name())
 		if err != nil {
 			logger.Errorf("failed to remove file %q. %v", assimilateConfPath.Name(), err)
 		}
+		err = os.Remove(outFilePath)
```

Comment:
we have created a single file why removing it twice

Reply:
There are actually two files: an input file, and an output file. Only the input file was previously deleted, so now the output file is as well.

Reply:
Got it